authorpromote.Rmd

```diff
@@ -10,14 +10,15 @@ Blog posts then live a life of their own: some people will still access them, e.
 
 When your post is published, here is what we do to promote it.
 
-- Tweet from [rOpenSci](https://twitter.com/rOpenSci) with a link to your post, tagging post author(s) and package reviewers with a link to the software peer review thread where relevant.
-rOpenSci has more than 25,000 followers on Twitter.
+- Post from [rOpenSci Mastodon account](https://hachyderm.io/@rOpenSci) and [LinkedIn page](https://www.linkedin.com/company/ropensci/) with a link to your post, tagging post author(s) and package reviewers with a link to the software peer review thread where relevant.
+rOpenSci has more than 2,000 followers on Mastodon and more than 1,000 on LinkedIn.
 
-- List your post, with author names, in our [biweekly newsletter](https://ropensci.org/news). 
-Our newsletter has more than 2,000 email subscribers and additional readers through RSS & JSON feeds.
+- List your post, with author names, in our [monthly newsletter](https://ropensci.org/news).
 
-- Our blog and tech notes are syndicated to and tweeted by [R-bloggers](https://www.r-bloggers.com/) and [R Weekly](https://rweekly.org/). R-Bloggers _cross-posts_ posts so it adds page views we can't measure, whereas R Weekly drives traffic to blog posts on our website. R-Bloggers had nearly 80,000 Twitter followers in February 2020.
+Our newsletter has more than 2,500 email subscribers and additional readers through RSS & JSON feeds.
```

Comment:
I feel the number is misleading. Our newsletter is opened by more than 1,000 readers each month?

Reply:
We can also say we have 2,500 subscribers with a 50% open rate.   Which is more accurate? But I'm ok with only mention de 1,000 readers.

Reply:
I think it's just easier to mention the readers!